apps/vx-mark/frontend/src/app_root.tsx

```diff
@@ -757,6 +718,15 @@ export function AppRoot({
     storage,
     initializedFromStorage,
   ]);
+  if (!machineConfigQuery.isSuccess) {
+    return (
+      <UnconfiguredScreen
+        hasElectionDefinition={Boolean(optionalElectionDefinition)}
+      />
```

Comment:
I'm wondering whether this is the right screen to display for this case since it's really just an ephemeral loading state and not a true unconfigured state. Something like [VxScan's loading configuration screen](https://github.com/votingworks/vxsuite/blob/main/apps/vx-scan/frontend/src/screens/loading_configuration_screen.tsx) might make more sense

Reply:
I think rendering `null` is also not a bad choice for queries that we expect to load extremely quickly like this one